conans/util/dates.py

```diff
@@ -31,14 +31,18 @@ def timestamp_to_str(timestamp):
 
 
 def timedelta_from_text(interval):
-    match = re.search(r"(\d+)([mhd])", interval)
+    match = re.search(r"(\d+)([smhdw])", interval)
     try:
         value, unit = match.group(1), match.group(2)
```

Comment:
the overall code could be significantly simplified in order to avoid if spagetti:
```
name = {'s': 'seconds',
        'm': 'minutes',
        'h': 'hours',
        'd': 'days',
        'w': 'weeks'}[unit]

print(datetime.timedelta(**{name: value}))
```

Reply:
Thank you, I was looking for it!

Reply:
Okay, added to current code.